spec/controllers/queue_items_controller_spec.rb

```diff
@@ -123,14 +123,21 @@
   describe "POST update_queue" do
     it_behaves_like "requires sign in" do
       let(:action) do
-        post :update_queue, queue_items: [{id: 2,  position: 3}, {id: 1, position: 1}]
+        post :update_queue,
+          queue_items: [{id: 2,  position: 3}, {id: 1, position: 1}]
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Unnecessary spacing detected.<br>Space inside { missing.<br>Space inside } missing.